test/requests.test.js

```diff
@@ -242,6 +242,40 @@ describe("narratives", () => {
   }
 });
 
+describe("groupOverview", () => {
+
+  describe("groups", () => {
+    testPaths([
+      { case: "overview", path: "/groups/test/settings/overview" },
+    ]);
+  });
```

Comment:
These tests now fail after rebasing to include d36f8441e0fd7d9d288cd92aa898caeb356b8a9d and use the `authz.tags.Type.Group` on the overview endpoint, without conditionally adding `authz.tags.Visibility.Public` here:

https://github.com/nextstrain/nextstrain.org/blob/d36f8441e0fd7d9d288cd92aa898caeb356b8a9d/src/groups.js#L112-L114

`Visibility.Public` seems intended for datasets/narratives, so this is probably fine. I'll remove these tests given manual tests cover the same thing with authz.